distribution/openrct2.d.ts

```diff
@@ -3807,6 +3807,12 @@ declare global {
          */
         readonly guestGenerationProbability: number;
 
+        /**
+         * Spawns a new guest at the edge of the map.
```

Comment:
```suggestion
         * Spawns a new guest at a random peep spawn point.
```
This is technically more accurate.